src/conversion/mapper.js

```diff
@@ -307,11 +309,10 @@ export default class Mapper {
 	 *		<b>foo</b>              -> 3 // Length of an element which is not mapped is a length of its children.
 	 *		<div><p>x</p><p>y</p>   -> 2 // Assuming that <div> is not mapped and <p> are mapped.
 	 *
-	 * @private
 	 * @param {engine.view.Element} viewNode View node.
```

Comment:
I found it useful to use in `List` feature converters. It looks like function like this may be useful in creating more complicated converters. After all this is not some kind of magic, it just returns view element's length in model. It's even easy to describe.

https://github.com/ckeditor/ckeditor5-list/blob/t/1b/src/converters.js#L542